docs/index.md

```diff
@@ -27,7 +27,8 @@ The provider needs to be configured with the proper credentials before it can be
 terraform {
   required_providers {
     cdp = {
-      source = "registry.terraform.io/cloudera/cdp"
+      source = "cloudera-labs/cdp"
```

Comment:
I thought moving to the cloudera-labs namespace is temporary and for testing the publishing only. Is that not the case? In `main.go` we are still binding to `registry.terraform.io/cloudera/cdp`.